src/network/MissionControlProtocol.h

```diff
@@ -61,6 +61,8 @@ class MissionControlProtocol : public WebSocketProtocol { // TODO: add documenta
 	void handleDriveRequest(const json& j);
 	void sendJointPositionReport(const std::string& jointName, int32_t position);
 	void sendCameraStreamReport(const CameraID& cam, const std::string& b64_data);
+	void sendRoverPos();
+	void roverPosReportTask();
```

Comment:
We wanted to rename `roverPosReportTask` to `telemReportTask`, so this method declaration shouldn't be here. Just deleting this line should be fine.